changelog.md

```diff
@@ -2,6 +2,9 @@
 
 ## NEXT
 
+- Improved the speed of the `cai_causal_graph.causal_graph.CausalGraph.from_adjacency_matrix` and
+  `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph.from_adjacency_matrices` methods by introducing a
+  `validate` flag. When `False` (default), there are no checks for cycles.
```

Comment:
Given this changes prior behavior, I think we should release a minor tag. Can make a `0.6.0dev` tag in near future for @tadhgokeeffe to use on McCann and then stable when we are closer to `2024.9` release.

Reply:
I agree with releasing a minor tag

Reply:
Actually, will default to True and release a minor tag

Reply:
Refreshed myself on the problem @steven-causalens , I have a notebook that benchmarks this problem already so will run that as soon as we have the dev tag and see if speed up is enough